addons/mail/static/src/components/emoji/emoji.xml

```diff
@@ -2,9 +2,11 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.Emoji" owl="1">
-        <span class="o_Emoji p-1 fs-5 o_cursor_pointer" t-on-click="props.emojiView.emojiListView.onClickEmoji" t-att-title="props.emojiView.emoji.description" t-att-data-source="props.emojiView.emoji.sources[0]" t-att-data-unicode="props.emojiView.emoji.unicode" t-attf-class="{{ className }}" t-ref="root">
-            <t t-esc="props.emojiView.emoji.unicode"/>
-        </span>
+        <t t-if='props.emojiView.emoji.emojiCategories.includes(messaging.emojiRegistry.currentCategory)'>
```

Comment:
This `t-if` should not exist here.

It's `EmojiShowcase` that has a computed field `emojiViews`, and itself should compute the `EmojiView`s based on selected category and all emojis (both determine the subset of all emojis in this category).